### DISCUSSION TITLE
Timer feature

### DISCUSSION THREAD

**Discussion by barbatrukko** [sbidy/wiz_light#176] [Ideas]
In other similar device (like Meross, or other) I can select a timer for the bulb/strip and set that bulb poweroff automatically X minutes after poweron. Is nice for kids room.
I talked with WIZ support and now this option is not available ( I can create a program and select time to start and time to stop, but not X min after start).
I would create a function in Home assistant for trigger power off, but would be nice to add in this wiz integration.

**Comment by sbidy** [comment]
Mh, the UDP interface (which is used by this integration) doesn't have this feature. So you have to create a function in HASS.
